### PR TITLE
Merge client-side form feature branch

### PR DESCRIPTION
**Pull request by haijian-vaadin**

### CODE REVIEW

File: flow-server/pom.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
         <javaparser.version>3.15.1</javaparser.version>
         <swagger.codegen.version>3.0.2</swagger.codegen.version>
         <swagger.codegen.generators.version>1.0.2</swagger.codegen.generators.version>
+        <skipTests>true</skipTests>
```
Comment (manolo) [inline flow-server/pom.xml:24]: remove this, we had this for speeding up, but cannot go to master

File: flow-tests/pom.xml [hunk 2]
```diff
@@ -23,6 +23,7 @@
             concurrent running of test modules. -->
         <server.port>8888</server.port>
         <server.stop.port>8889</server.stop.port>
+        <skipTests>true</skipTests>
```
Comment (manolo) [inline flow-tests/pom.xml:26]: remove this

File: flow-server/pom.xml (hunk 1 above)
Comment (haijian-vaadin) [reply flow-server/pom.xml]: oh, thanks for catching this!

File: flow-tests/pom.xml (hunk 2 above)
Comment (haijian-vaadin) [reply flow-tests/pom.xml]: done

File: flow-server/pom.xml [hunk 3]
```diff
@@ -21,7 +21,6 @@
         <javaparser.version>3.15.1</javaparser.version>
         <swagger.codegen.version>3.0.2</swagger.codegen.version>
         <swagger.codegen.generators.version>1.0.2</swagger.codegen.generators.version>
-        <skipTests>true</skipTests>
```
Comment (manolo) [inline flow-server/pom.xml]: Couple of tests need to be adjusted

**Pull request merged**